core/src/main/java/com/yahoo/ycsb/DB.java

```diff
@@ -41,97 +41,94 @@
  * target application.  For the sake of comparison between experiments we also 
  * recommend you explain the semantics you chose when presenting performance results.
  */
-public abstract class DB
-{
-	/**
-	 * Properties for configuring this DB.
-	 */
-	Properties _p=new Properties();
+public abstract class DB {
+  /**
+   * Properties for configuring this DB.
+   */
+  private Properties properties = new Properties();
```

Comment:
We can't leave _p since that is not an allowed variable name by checkstyle. I don't see a way around this.